This fails to build for me:

```
go.mozilla.org/sops/config imports
        github.com/mozilla-services/yaml tested by
        github.com/mozilla-services/yaml.test imports
        go.mozilla.org/yaml.v2: cannot find module providing package go.mozilla.org/yaml.v2
make: *** [Makefile:22: vendor] Error 1
==> ERROR: A failure occurred in build().
```

--- a/sops/PKGBUILD
+++ b/sops/PKGBUILD
@@ -19,9 +19,7 @@ prepare() {
 
 build() {
   cd src/go.mozilla.org/sops
-  go mod download
-  make vendor
-  env GOPATH="${srcdir}" go build \
+  env GO111MODULE=on GOPATH="${srcdir}" go build \
     -asmflags all="-trimpath=${PWD}" \
     -gcflags all="-trimpath=${PWD}" \
     -ldflags all="-extldflags=${LDFLAGS}" \

--- a/sops/PKGBUILD
+++ b/sops/PKGBUILD
@@ -19,6 +19,7 @@ prepare() {
 
 build() {
   cd src/go.mozilla.org/sops
+  go mod download
   make vendor
   env GOPATH="${srcdir}" go build \
     -asmflags all="-trimpath=${PWD}" \

